fda-model/stageDamage/ImpactAreaStageDamage.cs

```diff
@@ -280,12 +280,14 @@ public List<UncertainPairedData> Compute(IProvideRandomNumbers randomProvider)
             List<UncertainPairedData> results = ConsequenceDistributionResults.ToUncertainPairedData(allStagesAtIndexLocation, consequenceDistributionResults);
             return results;
         }
-        public float[] ExtrapolateFromAboveAtIndexLocation(float[] stagesAtStructuresHighestProfile, float upperInterval, int stepCount)
+        public static float[] ExtrapolateFromAboveAtIndexLocation(float[] stagesAtStructuresHighestProfile, float upperInterval, int stepCount)
         {
             float[] extrapolatedStages = new float[stagesAtStructuresHighestProfile.Length];
+            int i = 0;
             foreach (float structureStage in stagesAtStructuresHighestProfile)
             {
-                extrapolatedStages[stepCount] = structureStage + upperInterval * stepCount;
+                extrapolatedStages[i] = structureStage + upperInterval * stepCount;
```

Comment:
bug fix

Reply:
refactor to be a regular for loop for(int i = 0; i < extropolated stages...... and so on. The combo for loop and counter isn't very clean, and the fix is super easy.

Reply:
done